blackbird_python/blackbird/listener.py

```diff
@@ -320,16 +323,21 @@ def exitArrayvar(self, ctx: blackbirdParser.ArrayvarContext):
             shape = tuple([int(i) for i in ctx.shape().getText().split(",")])
 
         value = []
+        parameters = []
+        rows = 0
```

Comment:
(Minor suggestion): maybe `array_rows` is slightly more descriptive?
```suggestion
        array_rows = 0
```